02 - Community Expansions/02.03 CSS Snippets by Category/Mara-Li Folder Color.md

```diff
@@ -0,0 +1,1176 @@
+*/
+
+.nav-folder-title[data-path^="00."]:hover {
+  background-color: var(--bg00) !important;
+  filter: invert(50%) sepia(100%) saturate(0%) hue-rotate(250deg) brightness(0%);
+  color: var(--F00) !important;
+}
+
+.nav-file-title[data-path^="00."]:hover,
+.nav-folder-title[data-path*="01."]:hover,
+.nav-folder-title[data-path*="02."]:hover,
+.nav-folder-title[data-path*="03."]:hover {
+  color: var(--F00) !important;
+  background-color: var(--bg00) !important;
+  filter: invert(50%) sepia(100%) saturate(0%) hue-rotate(250deg) brightness(0%);
+}
+
+.nav-folder-title[data-path^="10."]:hover,
+.nav-file-title[data-path^="10."]:hover {
+  color: var(--F10) !important;
+  background-color: var(--bg10) !important;
+  filter: saturate(150%);
+}
+
+.nav-folder-title[data-path^="20."]:hover,
+.nav-file-title[data-path^="20."]:hover,
+.nav-file-title[data-path^="20."][data-path*="30."]:hover,
+.nav-file-title[data-path^="20."][data-path*="10."]:hover,
+.nav-file-title[data-path^="20."][data-path*="20."]:hover {
+  color: var(--F20) !important;
+  background-color: var(--bg20) !important;
+  filter: saturate(150%);
+}
+
+.nav-folder-title[data-path^="30"]:hover,
+.nav-file-title[data-path*="30."]:hover {
+  color: var(--F30) !important;
+  background-color: var(--bg30) !important;
+  filter: saturate(150%);
+}
+
+.nav-folder-title[data-path^="40"]:hover,
+.nav-file-title[data-path*="40."]:hover {
+  color: var(--F40) !important;
+  background-color: var(--bg40) !important;
+  filter: saturate(150%);
+}
+
+.nav-folder-title[data-path^="50"]:hover,
+.nav-file-title[data-path*="50."]:hover {
+  color: var(--F50) !important;
+  background-color: var(--bg50) !important;
+  filter: saturate(150%);
+}
+
+.nav-folder-title[data-path^="60."]:hover,
+.nav-file-title[data-path*="60."]:hover {
+  color: var(--F60) !important;
+  filter: saturate(150%);
+  background-color: var(--bg60) !important;
+}
+
+.nav-folder-title[data-path^="ZZ."]:hover,
+.nav-file-title[data-path*="ZZ."]:hover {
+  color: var(--FZZ) !important;
+  filter: saturate(150%);
+  background-color: var(--bgZZ) !important;
+}
+
+.nav-folder-title[data-path^="Ressource"]:hover,
+.nav-file-title[data-path*="Ressource"]:hover {
+  color: var(--FZZ) !important;
+  filter: saturate(150%);
+  background-color: var(--bgZZ) !important;
+}
+```
+
+- [Folder Color [Background]](https://raw.githubusercontent.com/Mara-Li/Obsidian-Snippet-collection/main/folder-color_bg.css)
+- [Folder Color [Font]](https://github.com/Mara-Li/Obsidian-Snippet-collection/blob/main/folder-color_font.css)
+
+![[Mara_FolderColor.png]]![[Mara_FolderColor2.png]]
+
+*Note : The files and folder icons is from [[obsidian-icon-folder|Icon Folder]]*
+
+<u>Credit</u>: 
+- [Lithou](https://forum.obsidian.md/t/adding-color-to-obsidian-a-rainbow-of-possibility/12805/11)
+- [Javalent](https://github.com/valentine195/Obsidian-Vault/blob/master/.obsidian/snippets/colors.folders.css)
```

Comment:
I recommend moving this up above the css so people can quickly see what the snippet looks like since the css is so long on the page. Maybe somewhere around line 20